### ISSUE TITLE
Remove usage of  `zerolog.Logger.Msgf()` from zot code

### ISSUE THREAD

**Issue by andaaron** [project-zot/zot#1367]
zerolog.Logger.Msgf() customized log messages could be hard to parse by specialized logging tools.
In case we want to include variable names/values in the log messages, we want to add them in the log record separately.

For example:
```golang
zerolog.Logger.Msgf("Some log message %s", stringVariable)
```
would be replaced by
```golang
zerolog.Logger.Str("string variable name", stringVariable).Msg("Some log message")
```

We should also be consistent in the positioning of these variables in similar log messages.

**Comment by rchincha** [comment]
> "We should also be consistent in the positioning of these variables in similar log messages."
Not that important since it is a JSON map and fields can be out of order.

https://logz.io/blog/golang-logs/
^ what it gets us!

**Comment by nicoldr** [comment]
Issues solved in #1382
